WzComparerR2/CharaSimControl/ItemTooltipRender2.cs

```diff
@@ -262,6 +262,17 @@ private Bitmap RenderItem(out int picH)
                 picH = 10;
             }
 
+            if (sr["fixWidth"] != null)
+            {
+                //重构大小
+                g.Dispose();
+                tooltip.Dispose();
+
+                tooltip = new Bitmap(Int32.Parse(sr["fixWidth"]), DefaultPicHeight);
```

Comment:
Better to refactor all of these checks? There're total of 3 times creating new Bitmap object, too many GC memory costs.